### ISSUE TITLE
GraphQL - Hasura

### ISSUE THREAD

**Issue by Maxximus007** [drejohnson/sveltekit-kratos#1]
Hi, was looking at your project, and it's quite interesting! Since you ave planned GraphQL/Hasura, you might want to take a look at [Houdini](https://github.com/AlecAivazis/houdini). Perhaps a combination is feasible?

**Comment by drejohnson** [comment]
Yeah I haven't had a chance to play with Houdini yet, but it's definitely on my radar
